Logger/Logger.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+#include <fstream>
+
+#include "Logger.hpp"
+
+void Logger::log(std::string msg)
+{
+    if (logs.good())
```

Comment:
czy to good() jest odpowiednikiem is_open() ?

Tutaj zachowanie powinno być takie moim zdaniem:
1. Jeśli plik nie byl otwarty
- otwórz plik
- zapisz loga
2. W przeciwnym wypadku
- zapisz loga


Fajnie, że destruktor zamyka plik, ale teraz mamy sytuacje, gdzie plik jest zamykany raz, a otwierany wiele razy przy kazdym uzyciu  metody log